### DISCUSSION TITLE
Show message content in replying to ... textbox

### DISCUSSION THREAD

**Discussion by anqueue** [Chatterino/chatterino2#3896] [Ideas] [answered]
In webchat whenever you are replying to someone's message it shows the message content while in Chatterino ([ebc7852f](https://github.com/Chatterino/chatterino2/commit/ebc7852f)) it does not. This can cause confusion on what message from the author you are replying to:
![Chatterino and webchat comparison](https://user-images.githubusercontent.com/35087590/182048881-605e17bd-6995-44f8-8ab6-0795d1f9a4f9.png)

**Comment by Felanbird** [comment, marked as answer]
Agreed, I remember running into this while testing it, but I believe I just pushed it to the side in my brain.
